Contributing.md

```diff
@@ -28,6 +28,10 @@ Ground Rules for Development
 
     When making commits you should include the issue number in the commit, this allows the commits to be linked the git issue. For example if I was working under my branch '63-Backend-SignUp' and a validator was implemented as part of the feature. The commit would look like '[#63] Add Validator for Input Fields'. 
 
+- For backend, before creating your pull request, please run the following command - it will create the javascript files from the typescript.
```

Comment:
shouldn't this be automated? part of the npm run start script?

Reply:
I had issues when I tried to integrate with nodemon

Reply:
i guess lets discuss if we wanna keep this at all